### PR TITLE
Update the README.md file to describe the repo and add links to personal profiles

### PR DESCRIPTION
**Pull request by dhruvi618**

### CODE REVIEW

**Comment by dhruvi618** [comment]
@

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,14 +2,27 @@
...
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Dhruvi's Portfolio</h1>
+      <p>This is Dhruvi's portfolio.</p>
+      <p>Hi there! My name is Dhruvi and I am a STEP intern @ Google! I study CS at Rutgers University-New Brunswick!</p>
+      <div class="icons"> 
+            <a href="https://www.linkedin.com/in/dhruvi-sheth/">
+                <img alt="Linkedin" src="images/linkedin.png" width="30" height="25">
```
Comment (tcwanggoogle) [inline portfolio/src/main/webapp/index.html:16]: nit: Any particular reason not to make it a square?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,14 +2,27 @@
...
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Dhruvi's Portfolio</h1>
+      <p>This is Dhruvi's portfolio.</p>
+      <p>Hi there! My name is Dhruvi and I am a STEP intern @ Google! I study CS at Rutgers University-New Brunswick!</p>
+      <div class="icons"> 
+            <a href="https://www.linkedin.com/in/dhruvi-sheth/">
```
Comment (tcwanggoogle) [inline portfolio/src/main/webapp/index.html:15]: super tiny nit: "/" is inconsistent with the other URL

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (dhruvi618) [reply portfolio/src/main/webapp/index.html:16]: only for aesthetic reason since the logo itself is not a square

Comment (tcwanggoogle) [reply portfolio/src/main/webapp/index.html:16]: Thanks for the clarification

**Comment by kmh287** [comment]
LG. To add on to Tiancong's comment, the title should be a concise summary of the discrete unit of functionality added in the PR. The description is your opportunity to provide a more detailed enumeration of all the changes in the PR.

**Pull request merged**